config/locales/application_form.yml

```diff
@@ -436,11 +440,6 @@ en:
             end_date:
               invalid: Enter an end date in the correct format
               in_the_future: Enter an end date that is not in the future
-        candidate_interface/work_breaks_form:
-          attributes:
-            work_history_breaks:
-              blank: Tell us about any breaks in your work history
-              too_long: Explanation of any breaks in your work history must be %{count} characters or fewer
```

Comment:
`too_long` is not actually called. [Created a related card](https://trello.com/c/nCljMNvt/562-error-for-word-count-should-include-maximum-word-count)

Reply:
Wondering if we need a character limit _and_ a word limit?